src/lib.rs

```diff
@@ -188,19 +188,30 @@ fn is_supported_content(content_type: &str) -> bool {
   supported.contains(&content_type)
 }
 
+fn get_content_type(response: &Response) -> &str {
```

Comment:
Getting the content type (without the charset) is a little more length in reqwest, thus this function.

Reply:
I was thinking about this and decided to check how `ureq` did it since their method returned a plain `str`. You can find the implementation [here](https://docs.rs/ureq/2.5.0/src/ureq/response.rs.html#198-207) but basically they just set their default to `text/plain`, I'll just do the same. 

Thanks for the heads up!